extensions/ql-vscode/src/queryserver-client.ts

```diff
@@ -168,4 +178,24 @@ export class QueryServerClient extends DisposableObject {
       delete this.progressCallbacks[id];
     }
   }
+
+  /**
+   * Updates the active query every time there is a new request to compile.
+   * The active query is used to specify the side log.
+   *
+   * This isn't ideal because in situations where there are queries running
+   * in parallel, their output is interleaved, but because the activeQuery
+   * is fixed, the previous query will have results mixed in with the active
+   * query's side log.
+   *
+   * The ideal solution would be to extract the active query from the log
```

Comment:
I think can simplify and update this comment to reflect that in situations where there are queries running in parallel, the logs that are emitted are fundamentally not easily assignable to individual queries.

Reply:
Oops. Should not be there.

Reply:
Will do.